server/appServices.js

```diff
@@ -282,6 +306,64 @@ async function watchMobileSecurityApps(namespace, kubeclient) {
   });
 }
 
+function handleUPSEventData(event, suffix, namespace, kubeclient) {
+  if (event.type === 'ADDED' && event.object && event.object.metadata.name.endsWith(suffix) && !updating) {
+    const data = getApp(appCollect, getAppName(event));
+
+    const currentServices = data.status.services.filter(ser => ser.name === 'push');
+    const service = currentServices[0];
+
+    if (event.object.kind === 'AndroidVariant') {
+      if (service && service.config && typeof service.config.android !== 'undefined') {
```

Comment:
why 
> if (service && service.config && **typeof** service.config.android **!== undefined**) {

instead of 

> if (service && service.config && service.config.android) {

 ?

Reply:
Good question, I guess it don't need to be that way.